diff --git a/lib/main.dart b/lib/main.dart
index 89193d1..8f664ce 100644
--- a/lib/main.dart
+++ b/lib/main.dart
@@ -83,7 +83,9 @@ void main() async {
     // 这一行会导致全屏的时候抖3次
     // await windowManager.setAsFrameless();
 
-    await windowManager.setHasShadow(true);
+    if (!Platform.isLinux) {
+      await windowManager.setHasShadow(true);
+    }
     // 设置窗口位置
     final prefs = await SharedPreferences.getInstance();
     if (prefs.containsKey('window_x') && prefs.containsKey('window_y')) {
